prow/cmd/deck/main.go

```diff
@@ -228,7 +228,15 @@ func prodOnlyMain(o options, mux *http.ServeMux) {
 			Endpoint:     github.Endpoint,
 		}
 
-		userDashboardAgent := userdashboard.NewDashboardAgent(&githubOAuthConfig, logrus.WithField("client", "user-dashboard"))
+		var repos []string
+		for k := range configAgent.Config().Presubmits {
+			repos = append(repos, k)
+		}
```

Comment:
In addition to including repos for which we have presubmit jobs, we should also include repos that have Tide queries (even if they don't have prow jobs).

Reply:
done